pegasus_wrapper/resource_request.py

```diff
@@ -105,13 +116,38 @@ def unify(self, other: ResourceRequest) -> ResourceRequest:
             num_gpus=other.num_gpus if other.num_gpus is not None else self.num_gpus,
         )
 
+    def convert_time_to_slurm_format(self, job_time_in_minutes: int) -> str:
+        hours, mins = divmod(job_time_in_minutes, 60)
+        return f"{hours}:{str(mins)+'0' if mins < 10 else mins}:00"
+
+    def check_time_limits(self) -> None:
```

Comment:
This ties the code too tightly to our current cluster configuration. I'd suggest instead documenting in the README what sort of errors the user will see if they specify too high a time limit.